src/wiseagents/agents/coordinator_wise_agents.py

```diff
@@ -101,6 +101,44 @@ def set_response_delivery(self, response_delivery: Callable[[], WiseAgentMessage
         """Set the function to deliver the response to the client."""
         self._response_delivery = response_delivery
 
+class SequentialMemoryCoordinatorWiseAgent(SequentialCoordinatorWiseAgent):
+    yaml_tag = u'!wiseagents.agents.SequentialMemoryCoordinatorWiseAgent'
+    
+    def __new__(cls, *args, **kwargs):
+        """Create a new instance of the class, setting default values for the instance variables."""
+        obj = super().__new__(cls)
+        obj._system_message = None
+        return obj
+
+    def __init__(self, name: str, description: str, transport: WiseAgentTransport, agents: List[str], system_message: Optional[str] = None):
+        self._system_message = system_message
+        super().__init__(name, description, transport, agents)
+
+    @property
+    def system_message(self) -> str:
```

Comment:
Last small comment, we can remove this method and pass the system message in the super().__init__ call above.